src/test/java/org/eclipse/yasson/Jsonbs.java

```diff
@@ -22,4 +25,32 @@ public class Jsonbs {
 	public static final Jsonb nullableJsonb = JsonbBuilder.create(new JsonbConfig().withNullValues(Boolean.TRUE));
     public static final YassonJsonb yassonJsonb = (YassonJsonb) JsonbBuilder.create();
 	public static final YassonJsonb bindingYassonJsonb = (YassonJsonb) new JsonBindingProvider().create().build();
+
+	private static void testWithJsonb(Supplier<Jsonb> supplier, Consumer<Jsonb> consumer){
```

Comment:
Why have you decided to create these methods? :-)

Reply:
In the tests, one always need a Jsonb object to test it. So there are these repeating parts in the test which creates this object one way or another. Sometimes it is in the try-with-resources block, sometimes one forgot about this.

So I extracted these repeating parts into the methods. These methods are the fixtures for the test. The test gets the Jsonb over the consumer and only have to implement the test logic itself in the consumer. No need to write this boilerplate code over and over again, with some unneeded combinations.

I choose to place these methods into this class because this class already has some common constants and methods which used all over the test code. If you think we need another class just for these static methods - I can do it.